src/components/CustomerQuota/ItemsSelection/IdentifierLayout/IdentifierSelectionInput.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FunctionComponent, useState, useEffect } from "react";
+import { View } from "react-native";
+
+import { size } from "../../../../common/styles";
+import { useTranslate } from "../../../../hooks/useTranslate/useTranslate";
+import { PolicyChoices } from "../../../../types";
+import { DropdownFilterInput } from "../../../DropdownFilterModal/DropdownFilterInput";
+import { sharedStyles } from "./sharedStyles";
+
+export const IdentifierSelectionInput: FunctionComponent<{
+  addMarginRight: boolean;
+  label: string;
+  onSelectDropdown: (choice: string) => void;
+  dropdownItems?: PolicyChoices[];
+}> = ({ addMarginRight, label, onSelectDropdown, dropdownItems = [] }) => {
+  const { c13nt } = useTranslate();
+  const [selectedChoice, setSelectedChoice] = useState<PolicyChoices>();
+
+  const onItemSelection = (item: PolicyChoices): void => {
+    setSelectedChoice(item);
+  };
+
+  useEffect(() => {
+    selectedChoice
+      ? onSelectDropdown(selectedChoice.value)
+      : onSelectDropdown("");
```

Comment:
What is the purpose of `onSelectDropdown`?

Reply:
it should trigger the generic `setInputValue` and `updateIdentifierValue` that are sued by other identifier inputs.